dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -170,10 +171,11 @@ internal open class RumViewScope(
             else -> delegateEventToChildren(event, writer)
         }
 
-        return if (isViewComplete()) {
-            null
+        if (isViewComplete()) {
+            GlobalRum.updateRumContext(getRumContext().copy(viewType = RumViewType.NONE))
```

Comment:
You should do that when stopping the current view only. The `isViewComplete()` can return true a long time after the view is stopped (e.g.: an activity goes in background, but there are ongoing network requests, so the view is still not complete). 
You could also introduce some issues if you navigate to another view, and 5 secondes later you update the rum context with your own...

Reply:
You're right, I am going to do the change here.